src/profile/webid.test.ts

```diff
@@ -459,24 +565,30 @@ describe("getPodUrlAll", () => {
 
     // Mock the two consecutive fetches to the WebID document and the profile
     //resource linked from it.
-    const mockedFetch = jest
-      .fn(fetch)
-      .mockResolvedValueOnce(
-        new Response(await triplesToTurtle(toRdfJsQuads(webIdProfile)), {
-          headers: {
-            "Content-Type": "text/turtle",
-          },
-          url: "https://some.profile",
-        } as ResponseInit)
-      )
-      .mockResolvedValueOnce(
-        new Response(await triplesToTurtle(toRdfJsQuads(altProfileAll[0])), {
-          headers: {
-            "Content-Type": "text/turtle",
-          },
-          url: "https://some.alt-profile",
-        } as ResponseInit)
-      );
+    // The WebID is fetched using the default fetch
+    const { fetch: mockedUnauthFetch } = jest.requireMock("cross-fetch") as {
+      fetch: jest.Mock<
+        ReturnType<typeof window.fetch>,
+        [RequestInfo, RequestInit?]
+      >;
+    };
+
+    mockedUnauthFetch.mockResolvedValueOnce(
+      new Response(await triplesToTurtle(toRdfJsQuads(webIdProfile)), {
+        headers: {
+          "Content-Type": "text/turtle",
+        },
+      })
+    );
+
+    const mockedFetch = jest.fn(fetch).mockResolvedValueOnce(
+      new Response(await triplesToTurtle(toRdfJsQuads(altProfileAll[0])), {
+        headers: {
+          "Content-Type": "text/turtle",
+        },
+        url: "https://some.alt-profile",
+      } as ResponseInit)
+    );
```

Comment:
Maybe in the expect section, you could add something like:

```js
expect(mockedFetch).toHaveBeenCalledTimes(1);
expect(mockedUnauthFetch).toHaveBeenCalledTimes(1);
```

Just to be more explicit about testing what was called?